lib/config.js

```diff
@@ -3,25 +3,24 @@ class Config {
   // baseUrl;
   // apiKey;
 
-  constructor(arg1, arg2) {
+  constructor (arg1, arg2) {
```

Comment:
I normally choose `standard` (or `semistandard`) in node because it's a zero-config linter + it also fixes the errors it finds.

All of our other linter files are for typescript...which we probably should convert this package to at some point. I just didn't want to waste time trying to write an eslint file.